backend/src/main/resources/static/asset-manifest.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
it seems like you have included your static frontend in this commit - generating the frontend build by hand can lead to errors down the road and should if possible be automated within your deployment CI/CD

Reply:
thx. done.